advocacy_docs/partner_docs/ThalesGuide/03-SolutionSummary.mdx

```diff
@@ -11,4 +18,4 @@ Thales CipherTrust Transparent Encryption (CTE) is designed to meet data securit
 
 
 !!! Note
-    EDB Postgres Extended with BDR (Bi-Directional Replication)
\ No newline at end of file
+    EDB Postgres Extended represents EDB Postgres Extended* with BDR (Bi-Directional Replication) and barman.
```

Comment:
@josh-heyer I think we need to include a closing !!! on the line following the text of this note. The note is being interpreted as a heading in the PDF.